doc/decisions/frontend_polygon_libraries.md

```diff
@@ -0,0 +1,46 @@
+# Polygon Libraries
+
+## Problem
+
+For drawing zones (shadings, hydrology, zones), we need to transform a set of points, drawn using a brush, into a polygon. Furthermore, we want to be able to create unions of polygons so that zones can be expanded.
```

Comment:
```suggestion
For drawing areas of shadings, hydrology, zones, etc., we need to transform a set of points, drawn using a brush, into a polygon. Furthermore, we want to be able to create unions of polygons so that areas can be expanded and differences of polygons to reduce areas.
```